resources/qml/WelcomePages/AddPrinterBySelectionContent.qml

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2019 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+
+import UM 1.3 as UM
+import Cura 1.1 as Cura
+
+
+//
+// This component contains the content for the "Add a printer" (network) page of the welcome on-boarding process.
+//
+Item
+{
+    UM.I18nCatalog { id: catalog; name: "cura" }
+
+    Label
+    {
+        id: titleLabel
+        anchors.top: parent.top
+        anchors.topMargin: 40
+        anchors.horizontalCenter: parent.horizontalCenter
+        horizontalAlignment: Text.AlignHCenter
+        text: catalog.i18nc("@label", "Add a printer")
+        color: UM.Theme.getColor("primary_button")
+        font: UM.Theme.getFont("large_bold")
+        renderType: Text.NativeRendering
+    }
+
+    DropDownWidget
+    {
+        id: addNetworkPrinterDropDown
+
+        anchors.top: titleLabel.bottom
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.margins: 20
+
+        title: catalog.i18nc("@label", "Add a networked printer")
+        contentShown: true  // by default expand the network printer list
+
+        onClicked:
+        {
+            if (contentShown)
+            {
+                addLocalPrinterDropDown.contentShown = false
+            }
+        }
+
+        contentComponent: networkPrinterListComponent
+
+        Component
+        {
+            id: networkPrinterListComponent
+
+            AddNetworkPrinterScrollView
+            {
+                id: networkPrinterScrollView
+
+                maxItemCountAtOnce: 6  // show at max 6 items at once, otherwise you need to scroll.
```

Comment:
Wait, so the default in the AddNetworkPrinterScrollView is not used at all? Maybe we need to change the default to 6 there and remove the override from here.

Reply:
The list height will be different for the network printer list and the local printer list because they use different delegate widgets plus the local list uses sections. IMO I think it's ok like this because it's explicit, and if you find the default value not being a good fit, you can always tweak it through this property.